Rakefile

```diff
@@ -303,6 +327,7 @@ namespace :install do
   tasks << :blockly_symlink
   tasks << :apps if CDO.build_apps
   tasks << :shared if CDO.build_shared_js
+  tasks << :code_studio
```

Comment:
So, we do have a [chef recipe that creates these symlinks](https://github.com/code-dot-org/code-dot-org/blob/staging/cookbooks/cdo-apps/recipes/dashboard.rb#L48) which I will need to update (adding a TODO to this PR description) but [the `local_environment?` check](https://github.com/code-dot-org/code-dot-org/blob/staging/Rakefile#L217) inside the `install:code_studio` task makes the install task a no-op in chef-configured environments anyway.  

For local development (or any non-chef environment) this is the only way I can find that we create those symlinks.  @wjordan any idea if I'm missing anything here?